### PR TITLE
Family page: Add pop-in animations

### PR DESCRIPTION
**Pull request by maureenlholland**
## One-line summary

Add pop-in animations on select elements in family page

## Significant changes and points to review

Browser boxes animate in their sections
Hero section: Tech talk and "The" box animate in
Mental health section: multiple browser stagger their pop-in animations
Private mode section: Background of browser changes to dark theme, private mask pops in (thanks to @nathan-barrett for the majority of work here!)

## Issue / Bugzilla link
https://github.com/mozilla/bedrock/issues/12111

## Testing

http://localhost:8000/firefox/family/

Scroll down page to see animations

WITH REDUCED MOTION pref
Scroll down page to see static content

⚠️ Do not merge pending design approval from demo

### CODE REVIEW

File: media/css/firefox/family/components/modules/_privacy.scss [hunk 1]
```diff
@@ -48,7 +48,7 @@
             }
 
             .c-content {
-                grid-row: 2;
+                grid-row: 2 / -1;
```
Comment (maureenlholland) [inline media/css/firefox/family/components/modules/_privacy.scss:51]: Not strictly related to this change, but there was a lot of extra space in the privacy section 2nd column, this makes the text in the 1st column span all subsequent rows, so it shouldn't create extra space between the browser and blurb
spacing-fix

**Comment by maureenlholland** [comment]
to-do:
- private mode should still have dark bg on reduced motion pref, but no animation

**Comment by maureenlholland** [comment]
note: this is in design review for the speed/bounciness of the animation, the only code that might change based on feedback from that is the keyframe declaration

**Review by alexgibson — APPROVED** [review]
Tested this locally and works well with animation, prefers-reduced-motion, and no-JS.

nice work! r+

**Comment by maureenlholland** [comment]
design approved! merging

**Pull request merged**